docs/src/pages/docs/Connecting to Databases/docker-add-drivers.mdx

```diff
@@ -58,6 +58,11 @@ After the rebuild is complete, which make take a few minutes, relaunch:
 ```
 docker-compose up
 ```
+If with the previous command, your docker gets up but Superset gets stuck loading the welcome screen, you can try with
```

Comment:
It's not trial and errors approach. There are different docker-compose files for a reason. If there is a problem with `docker-compose.yml` we should address that problem not trying non-dev docker-compose.

Reply:
Since no reply from OP, I incorporated your feedback :)